internal/operators/cnv/cnv_operator.go

```diff
@@ -274,3 +281,16 @@ func sanitizeID(id string) string {
 func getDeviceKey(vendorID string, deviceID string) string {
 	return vendorID + ":" + deviceID
 }
+
+// If CNV is deployed on SNO, we want at least one non bootable disk (i.e discoverable by LSO)
+// with certain size threshold for the hpp storage pool
+func validDiscoverableSNODisk(disks []*models.Disk, installationDiskID string) error {
+	for _, disk := range disks {
+		if (disk.DriveType == ocs.SsdDrive || disk.DriveType == ocs.HddDrive) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
+			if disk.SizeBytes > conversions.GibToBytes(discoverableDiskSizeThresholdGi) {
+				return nil
+			}
+		}
+	}
+	return fmt.Errorf("CNV with SNO requires a discoverable disk with %d Gi", discoverableDiskSizeThresholdGi)
```

Comment:
Not sure about the message here. What discoverable means? Is CNV something that the user knows about?

Reply:
Though I see `CNV` used in other messages ....

Reply:
But you are right: the user-facing term should be `OpenShift Virtualization`, and should be taken from a variable, to allow "whitelabeling" of assisted-installer for upstream OKD.

Reply:
Tweaked msg according to suggestion and changed the validation msgs from `CNV->OpenShift Virtualization`
Regarding whitelabeling assisted-service, would that be OK to not tackle in this PR?

Reply:
Sure, particularly if defining a `VirtProduct` constant would cause you to touch more modules. PRs should be concise.